src/hooks/useTextSelectionHandler.ts

```diff
@@ -27,15 +28,19 @@ export default function useTextSelectionHandler(
       }
     } else {
       const rect = selectionRect(selection)
+      const textRange = selectionTextRange(selection)
       const parentRect = containerRef.current.getBoundingClientRect()
+      console.log(selection)
       const rectInContainerCoordinates = {
         top: rect.top - parentRect.top,
         left: rect.left - parentRect.left,
         width: rect.width,
         height: rect.height,
         bottom: rect.bottom - parentRect.top,
         right: rect.right - parentRect.left,
+        ...textRange,
       }
+      console.log(textRange)
```

Comment:
Are we keeping the debug? also on line 33

Reply:
I missed this because I was tired